cypress/elements/fileMenu/open.js

```diff
@@ -2,21 +2,21 @@ import { generateRandomChar, generateRandomNumber } from '../../utils/random'
 import { clickMenuBarFileButton } from '../menuBar'
 import { FILE_MENU_BUTTON_OPEN, clickFileMenuButton } from '.'
 
-const openModalEl = '*[class^="MuiDialogContent"]' // TODO: Add data-test to open modal to target this better
+const openModalEl = 'open-file-dialog-modal'
 const openModalFooterEl = '*[class^="MuiTableFooter"]'
 const openModalToolbarEl = '*[class^="MuiToolbar-root"]'
 const createdByOthersEl = '[data-value=byothers]'
 const openModalItemContainerEl = '*[class^="MuiTableBody"]'
 
 const searchAOByName = name =>
-    cy.get(openModalEl).find('*[type="search"]').clear().type(name)
+    cy.getBySel(openModalEl).find('*[type="search"]').clear().type(name)
```

Comment:
preferably replace the `find('*[type="search"]'` with a `dataTest` prop for that input field and target that instead. the old selector was a bit of a hack because of MUI.